wordpress-mobile/WordPress-Android
# Upgrade Gradle wrapper, Bintray and enable BuddyBuild

Upgrades the gradle wrapper to `v3.5` and the Bintray plugin to `v0.5.0` to make the build compatible with BuddyBuild. Also, try to pick up the `gradle.properties` file from the BuddyBuild secret files folder if it exists and no such file is present in code tree's root already.

The rest of the BuddyBuild setup is performed from their website.

To test:
* Try out `./gradlew assemble` and notice it works fine and produces all the APKs (`vanilla`, `wasabi` and `zalpha`, all with `debug` and `release`)

@maxme or @mzorz , wanna pick this up for Review? Thanks!
